.github/workflows/publication_maven.yml

```diff
@@ -27,14 +39,14 @@ jobs:
         env:
           OSSRH_USER: ${{ secrets.ossrh_user }}
           OSSRH_PASSWORD: ${{ secrets.ossrh_password }}
-          OSSRH_STAGING_PROFILE_ID: ${{ secrets.ossrh_staging_profile_id }}
           PGP_KEYID: ${{ secrets.pgp_keyid }}
           PGP_KEY: ${{ secrets.pgp_key }}
           PGP_PASSWORD: ${{ secrets.pgp_password }}
         run: >
-          ./gradlew publishRelease -Pkaspresso.ossrh.user=${OSSRH_USER}
+          ./gradlew publishAllPublicationsTo${{ inputs.releaseType }}Repository
+          -Pversion=${{ inputs.version }}
```

Comment:
It would be more clear if all kaspresso related parameters start with `kaspresso.` prefix. Up to you here.

Reply:
done